README.md

```diff
@@ -0,0 +1,202 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>Covid 19 Tracker</b></h1>
+  <br/>
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Screenshot](#screenshot)
+  - [Loom Presentation Video](#loom-video)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Covid19 Tracker <a name="about-project"></a>
+
+
+["Covid19 Tracker"] :- "Covid19 Tracker" is a Project working with the real live data from the opendiseases API. the project provides report about the covid 19 from the world and it gives you the ability to filter the data by country. the project is built using react.js and redux toolkit and it is tested using jest and react testing library.
+
+## 🛠 Built With <a name="built-with">HTML,CSS and React and Redux</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+    <li><a href="https://legacy.reactjs.org/">React website</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux Toolkit website</a></li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML]**
+- **[CSS]**
+- **[Javascript]** ---> with ES6
+- **[Webpack]**
+- **[React]**
+- **[Redux]**
+- **[Jest_Test]**
+- **[Redux_Toolkit]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Screenshot <a name="screenshot"></a>
+
+[N/A]...
+
+## 💻 Loom Presentation Video <a name="loom-video"></a>
+
+- [Live Presentation Link](https://www.loom.com/share/27ee19a65ff9472caf87b81ced71fcf7)
+
```

Comment:
- [ ] In the new video you're presenting the project very nicely however you are only showing the page, please also present a piece of the code of which you are proud.

Reply:
Hi @mikethreels, I appreciate your suggestion, but I don't think it is part of the requirement, the requirement is like this `You should also highlight some interesting piece of code or something you built that you are very proud of. either you can demonstrate a piece of code or something I built that I am very proud of and I share what I am proud of, can you please review the requirements again, because I don't think it is compulsory to demonstrate a code and the page.